examples/2d/mesh2d_modify_live.rs

```diff
@@ -0,0 +1,104 @@
+//! Shows how to change the attributes of a polygonal [`Mesh`], in a 2D scene, over time.
+
+use bevy::{prelude::*, sprite::MaterialMesh2dBundle};
+
+const DIFF_CONST: f32 = 0.2; // How fast the diffusion happens
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .add_systems(Update, update_mesh_colors)
+        .run();
+}
+
+// This Component tracks the mesh and the initial vertex colors. We can query
+// for it using an ECS System on each update. By tracking the handle, we can
+// get the particular mesh from the asset storage and modify it.
+#[derive(Component)]
```

Comment:
I think in this example, the pattern will be clearer if we use a resource here.

Reply:
This is less obvious to me how to change. Are you saying just do a `meshes.get_one().expect()`?

Reply:
No, swap to a [Resource](https://docs.rs/bevy/latest/bevy/ecs/system/trait.Resource.html). We can hold onto the handle there, and modify it by reading the `ResMut`.

Reply:
Ah I see. Doesn't the resource need to be inserted like `app.insert_resource(...)` i.e. upon initialization? Seems like that approach requires handling Optional<Handle<Mesh>> rather than Handle<Mesh> in that case, no? I see how having a global scoped MeshPhonebook makes sense, but the overhead seems not worth it.

Please excuse the simple questions - I'm brand new to the Bevy env.

Reply:
No worries! Resources can be initialized after app initialization as well, generally via `Commands`.

While an `Option<Handle>` (or event the default null handle) would work, in this case I think that deferred setup via `commands.insert_resource` is the clearest :)